superset/migrations/versions/2023-04-19_11-23_4c5da39be729_migrate_treemap_chart.py

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""migrate_treemap_chart
+
+Revision ID: 4c5da39be729
+Revises: 07f9a902af1b
+Create Date: 2023-04-19 11:23:23.192064
+
+"""
+from alembic import op
+from sqlalchemy.dialects.mysql.base import MySQLDialect
+
+from superset.migrations.shared.migrate_viz import MigrateTreeMap
+
+# revision identifiers, used by Alembic.
+revision = "4c5da39be729"
+down_revision = "07f9a902af1b"
+
+
+def upgrade():
+    # Ensure `slice.params` and `slice.query_context`` in MySQL is MEDIUMTEXT
+    # before migration, as the migration will save a duplicate form_data backup
+    # which may significantly increase the size of these fields.
+    if isinstance(op.get_bind().dialect, MySQLDialect):
```

Comment:
@michael-s-molina has there been any update to the underlying model which would then make this redundant?

Reply:
@john-bodley This migration is referenced by a previous script that was created a while ago. Because of that, we need to keep this piece of code even if the models were updated later.